test/Main.purs

```diff
@@ -8,6 +8,7 @@ import Control.Monad.Eff.Console
 import Data.Either
 
 data Foo = Foo Number String | Bar Number | Quux (Array String) | Baz {a :: Maybe String, bq :: Number} String
+         | Corge (Array Char)
```

Comment:
looks good to me.

On Thu, Sep 10, 2015 at 7:19 PM, Phil Freeman <notifications@github.com>
wrote:

> In test/Main.purs
> <https://github.com/purescript/purescript-generics/pull/15#discussion_r39226453>
> :
>
> > @@ -8,6 +8,7 @@ import Control.Monad.Eff.Console
> >  import Data.Either
> >
> >  data Foo = Foo Number String | Bar Number | Quux (Array String) | Baz {a :: Maybe String, bq :: Number} String
> > +         | Corge (Array Char)
>
> Ah ok :) Hadn't heard that one before..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/purescript/purescript-generics/pull/15/files#r39226453>
> .
>